app/views/welcome/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>WELCOME TO MONSTER SHOP</h1>
```

Comment:
dynamic!

Reply:
@elguapogordo Technically this is static.